test/ace/http1/server_test.exs

```diff
@@ -77,7 +77,7 @@ defmodule Ace.HTTP1.ServerTest do
 
     assert_receive {:tcp, ^socket, response}, 1000
 
-    assert response == "HTTP/1.1 500 Internal server error\r\nconnection: close\r\ncontent-length: 2\r\n"
+    assert response == "HTTP/1.1 500 Internal server error\r\nconnection: close\r\ncontent-length: 21\r\n"
```

Comment:
length 21?

Reply:
This may be wrong. Not sure how to calculate it.

Reply:
it's the length of the body after the headers so in this case 0.